Usuario y Contraseña 2.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Registros
+{
+    public partial class Form_Sesion : Form
+    {
+        int a = 0;
+        public Form_Sesion()
+        {
+            InitializeComponent();
+        }
+
+        private void Label1_Click(object sender, EventArgs e)
+        {
+
+        }
+
+        private void Txtcontraseña_TextChanged(object sender, EventArgs e)
+        {
+            
+        }
+
+        private void Btnaccederusu_Click(object sender, EventArgs e)
+        {
+            if (txtcontraseñausu.Text == "9876")
```

Comment:
Asumo que esta quemada la contraseña, solo por ahora?

Reply:
Correcto...por ahora. Modalidad: Pruebas...

Reply:
Listo, agrega un TOD entonces para que sepas que eso se debe quitar despues
```suggestion
//TODO: Este codigo es para probar el flujo, una vez se realize la conección de base de datos, quitar el valor quemado
 if (txtcontraseñausu.Text == "9876")
 ....
```